management/src/clusterm/manager/client.go

```diff
@@ -93,20 +93,24 @@ func (c *Client) doGet(rsrc string) ([]byte, error) {
 }
 
 // PostNodeCommission posts the request to commission a node
-func (c *Client) PostNodeCommission(nodeName, extraVars string) error {
-	return c.doPost(fmt.Sprintf("%s/%s", PostNodeCommissionPrefix, nodeName), extraVars, nil)
+func (c *Client) PostNodeCommission(nodeName string, extraVars string, hostGroup string) error {
+	req := &APIRequest{
+		HostGroup: hostGroup,
+	}
+	return c.doPost(fmt.Sprintf("%s/%s", PostNodeCommissionPrefix, nodeName), extraVars, req)
 }
 
 // PostNodesCommission posts the request to commission a set of nodes
-func (c *Client) PostNodesCommission(nodeNames []string, extraVars string) error {
+func (c *Client) PostNodesCommission(nodeNames []string, extraVars string, hostGroup string) error {
 	req := &APIRequest{
-		Nodes: nodeNames,
+		Nodes:     nodeNames,
+		HostGroup: hostGroup,
 	}
 	return c.doPost(PostNodesCommission, extraVars, req)
 }
 
 // PostNodeDecommission posts the request to decommission a node
-func (c *Client) PostNodeDecommission(nodeName, extraVars string) error {
+func (c *Client) PostNodeDecommission(nodeName string, extraVars string) error {
```

Comment:
minor: the extra `string` after nodeName is not needed as both the args are of same type.